vars/tdr.groovy

```diff
@@ -0,0 +1,47 @@
+// Call when deployment (intg or staging) has finished. Use the delay to ensure that the AWS load balancer allows access to the new version you are deploying.
+def runEndToEndTests(int delaySeconds, String stage) {
+    String[] params = [
```

Comment:
Sorry, only just spotted this: we're using 2 space indents in groovy files - see the [frontend Jenkinsfile](https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/Jenkinsfile-deploy) for example.